include/soci/type-conversion.h

```diff
@@ -42,17 +47,15 @@ class conversion_into_type
     typedef typename type_conversion<T>::base_type base_type;
 
     conversion_into_type(T & value)
-        : into_type<base_type>(details::base_value_holder<T>::val_, ownInd_)
+        : into_type<base_type>(details::base_value_holder<T>::val_, details::base_value_holder<T>::ownInd_)
```

Comment:
Why do we need to use `details::` here, inside details namespace? AFAICS it shouldn't be necessary and omitting it would make the code shorter and more readable.